compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -66,16 +66,19 @@ import dotty.tools.dotc.core.quoted._
  *    ```
  *  to
  *    ```
- *    inline def foo[T1, ...](inline x1: X, ..., y1: Y, ....): Seq[Any] => Object = { (args: Seq[Any]) => {
+ *    inline def foo[T1, ...](inline x1: X, ..., y1: Y, ..., inline f1: X => Y): Seq[Any] => Object = { (args: Seq[Any]) => {
```

Comment:
Maybe add `f1` at the original form as well in the comment.

Reply:
I removed it and made it part of the comment below

Reply:
Seems good, nice.